packages/website/docs/auxiliary/Annotations.mdx

```diff
@@ -0,0 +1,312 @@
+import CodeBlock from '@theme/CodeBlock'
+import { PropsTable } from '@site/src/components/PropsTable'
+import { DocWrapper, XYWrapper } from '../wrappers'
+
+export const sampleData = [0, 3, 7, 5, 10, 4, 2, 8, 4, 0].map((d, i) => ({ x: (i + 1), y: d }))
+
+export const defaultProps = () => ({
+  name: 'Annotations',
+  configKey: 'annotations',
+  data: sampleData.slice(0, 4),
+  showContext: 'container',
+  height: 150,
+})
+
+The _Annotations_ component allows you to add informative text to your visualizations.
+It is compatible with both _XY_ and _Single_ containers.
+
+## Basic Configuration
+Include the `Annotations` component in your _Single_ or _XY_ container with an array of **items**.
+A basic example looks like:
+
+<XYWrapper
+  {...defaultProps()}
+  height={300}
+  data={sampleData.slice(1, 9)}
+  containerProps={{
+    yDomain: [0, 15]
+  }}
+  components={[
+    { name: 'Area', props: { x: (d, i) => d.x, y: d => d.y, curveType: 'linear' }, key: 'components' },
+  ]}
+  items={[
+    { x: 10, y: 10, content: { text: 'Chart Title', fontSize: 30 }},
```

Comment:
The title is a bit far to the left.

<img width="894" alt="SCR-20240404-lhqu" src="https://github.com/f5/unovis/assets/755708/f74820a0-238e-4cfe-85d4-41a49e3a341b">


I would make it `x: 12` to **optically** align these two text blocks:
<img width="624" alt="image" src="https://github.com/f5/unovis/assets/755708/0acb6158-1759-4df3-a6f2-6751fa4321a9">